src/lv_misc/lv_types.h

```diff
@@ -74,6 +74,12 @@ typedef uint32_t lv_uintptr_t;
 
 #define LV_UNUSED(x) ((void) x)
 
+#define LV_CONCAT_(x, y) x ## y
```

Comment:
Please change it to `_LV_CONCAT`.
We denote internal API names with `_` prefix.

Reply:
Done.

Reply:
Thanks!